05-23 18:01:21 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 18:02:53 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 18:05:08 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 18:11:50 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 18:20:57 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 18:21:37 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 18:24:17 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 18:51:16 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 19:32:42 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 09:58:26 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 10:02:26 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 10:06:00 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 10:08:17 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 10:49:57 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 10:56:47 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 11:07:13 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 11:35:04 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-24 11:37:36 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:89)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
